sdk/mobile-center/src/main/java/com/microsoft/azure/mobile/utils/ShutdownHelper.java

```diff
@@ -0,0 +1,24 @@
+package com.microsoft.azure.mobile.utils;
+
+import android.os.Process;
+import android.support.annotation.VisibleForTesting;
+
+/**
+ * Shutdown helper.
+ */
+public class ShutdownHelper {
+
+    @VisibleForTesting
+    ShutdownHelper() {
+        /* Hide constructor. */
+    }
+
+    public static void shutdown() {
```

Comment:
public is not used according to code analysis, either make it package private or document/suppress warning.